test/validations/coercions_test.rb

```diff
@@ -22,4 +22,20 @@
     result = Lotus::Validations::Coercions.coerce(Integer, '')
     result.must_equal nil
   end
+
+  it 'returns an empty string when an empty string is given to coerce to String' do
+    result = Lotus::Validations::Coercions.coerce(String, '')
+    result.must_equal ''
+  end
+
+  it 'returns nil when an array is given to coerce to a string' do
+    result = Lotus::Validations::Coercions.coerce(String, [])
+    # This is not ideal behaviour
+    result.must_equal '[]'
```

Comment:
[].to_s # => "[]"
{}.to_s # => "{}"

it isn't the ideal behaviour but it's the ruby default, maybe it should be like this.